aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -84,6 +84,13 @@ struct Vec256 {
     }
     return ret;
   }
+  Vec256<T> map2(const Vec256<T> other, T (*f)(T, T)) const {
```

Comment:
Not sure this is worth it. Definitely requires some profiling to make this doesn't affect performance. I'd not introduce this kind of abstraction, unless you're absolutely sure this makes no difference. This can be quite a bit of effort to figure out exactly. So, for now, I'd not do this.